src/main/webapp/components/BuildSnapshotContainer.js

```diff
@@ -19,7 +19,7 @@ export const BuildSnapshotContainer = React.memo((props) =>
      * @see <a href="www.robinwieruch.de/react-hooks-fetch-data">Fetching</a>
      */
     React.useEffect(() => {
-      fetch(SOURCE).then(res => setData(res.json()));
+      fetch(SOURCE).then(res => setData(res.json())).catch(setData([]));
```

Comment:
(suggestion for an improvement in a future PR): consider surfacing some sort of a high-level message that there was a problem fetching data instead of just setting an empty list

Reply:
+1, this is a good idea, perhaps some feedback on the UI would be applicable.